src/oc_erchef/apps/oc_chef_wm/test/chef_wm_actions_queue_monitoring_tests.erl

```diff
@@ -628,3 +629,40 @@ meck_conn_failure() ->
             {error,{conn_failed,undefined}}
         end).
 
+aliveness_test() ->
+    {foreach,
+     fun() ->
+             ok
+     end,
+     fun(_) ->
+             catch(meck:unload(oc_httpc))
+     end,
+     [
+      {"it's alive",
+       fun() ->
+            meck:new(oc_httpc),
```

Comment:
You can put this in the setup fun (the first fun after forearch) if you call it in each case.